README.md

```diff
@@ -10,21 +10,23 @@ Current `nectarchain` build uses `ctapipe` master version.
 
 Here is how you should install:
 ```
-git clone https://github.com/cta-observatory/nectarchain.git
+git clone git@github.com:cta-observatory/nectarchain.git
 cd nectarchain
 conda env create --name cta --file environment.yml
```

Comment:
Can we change the name of the environment from cta to nectarchain ?

Reply:
Sure !